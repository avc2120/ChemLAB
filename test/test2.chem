function main ()
{
	element Na(1,1,1);
	element Cl(12,12,12);
	molecule NaCl{Na, Cl};
}
