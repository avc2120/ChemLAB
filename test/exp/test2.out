2
3
ChemLAB